server/src/controllers/christmas-tree.es6

```diff
@@ -393,30 +393,24 @@ const checkPermitValid = permitExpireDate => {
  * @function generateRulesAndEmail - Private function to generate svg, png, and rules html of a permit and send out email
  * @param {Object} permit - permit object
  */
-const generateRulesAndEmail = permit => {
+const generateRulesAndEmail = permit =>
   permitSvgService
     .generatePermitSvg(permit)
-    .then(permitSvg => {
-      permitSvgService.generatePng(permitSvg).then(permitPng => {
-        permitSvgService
-          .generateRulesHtml(true, permit)
-          .then(rulesHtml => {
-            permit.permitUrl = paygov.createSuccessUrl(permit.christmasTreesForest.forestAbbr, permit.permitId);
-            let rulesText = htmlToText.fromString(rulesHtml, {
-              wordwrap: 130,
-              ignoreImage: true
-            });
-            sendEmail(permit, permitPng, rulesHtml, rulesText);
-          })
-          .catch(error => {
-            logger.error(error);
-          });
+    .then(() => Promise.all([
+      permitSvgService.generatePng(permit),
+      permitSvgService.generateRulesHtml(true, permit),
+    ]))
+    .then((permitPng, rulesHtml) => {
```

Comment:
This was my mistake. This should be an array dereference.
```
.then(([permitPng, rulesHtml]) => {
```

Reply:
oh, yes, I think I didn't mean to quite tag you for review yet, but I would love to put in some tests.